server/views/pages/goal/review/index.njk

```diff
@@ -28,7 +28,7 @@
                 Description
               </dt>
               <dd class="govuk-summary-list__value" data-qa="goal-{{ goalLoop.index }}-description-value">
-                {{ goal.title | formatMultilineText | safe }}
+                <span class="app-u-multiline-text">{{ goal.title }}</span>
```

Comment:
To be sure this works consistently, I've wrapped the text in a `<span>` element that has the desired style. This is so that I can force the start of the text to be immediately after the opening tag that contains the style.

EG: this renders extra line spaces above the text because there is a load of white space between the `<dd>` that contains the class, and the text:
```
              <dd class="govuk-summary-list__value app-u-multiline-text" data-qa="goal-{{ goalLoop.index }}-description-value">{{ goal.title }}</dd>
```

I could have "fixed" it by putting it all on one line like this, but it felt somewhat "hidden" and would be too easy for a dev in the future to do it on several lines (as we have it currently)
```
              <dd class="govuk-summary-list__value app-u-multiline-text" data-qa="goal-{{ goalLoop.index }}-description-value">
                {{ goal.title }}
              </dd>
```

Reply:
Makes sense - tested and works well!